spec/rspec/rails/matchers/send_email_spec.rb

```diff
@@ -0,0 +1,86 @@
+# frozen_string_literal: true
+
```

Comment:
```suggestion
```
I've once tested on a huge codebase, no noticeable difference with or without this directive **in specs**.